inc/targetticket.class.php

```diff
@@ -556,7 +556,7 @@ public function prepareInputForUpdate($input) {
                   $input['destination_entity_value'] = $input['_destination_entity_value_entity'];
                   break;
                default :
-                  $input['destination_entity_value'] = 'NULL';
+                  $input['destination_entity_value'] = 0;
```

Comment:
Just to be sure, 0 for entity is a valid ID (root entity).
Without "null", how do you differentiate between no value and root entity ?

If you are aware that this will default this setting to the root entity and it does not impact the plugin behavior then it's all good.

Reply:
The 1st case is when we use a specific entity "hardcoded" in the  target (here we have an entity ID).

The 2nd case is when we must use the default entity of a user selected from a question

The 3rd case is when we use the entity selected from a GLPI Object / Entity question

In all other cases, there is significant value to save. You may read PluginFormcreatorAbstractTarget::getEnumDestinationEntity() to see all possibles cases.